website/pages/docs/concepts/integrated-storage.mdx

```diff
@@ -0,0 +1,249 @@
+
+Nodes that are joined to a cluster can be specified as non-voters. A non-voting
+node has all of Vault's data replicated to it, but does not contribute to the
+quorum count. This can be used in conjunction with [Performance
+Standby](/docs/enterprise/performance-standby/) nodes to add read scalability to
+a cluster in cases where a high volume of reads to servers are needed.
+
+```shell
+$ vault operator raft join -non-voter https://node1.vault.local:8200
+```
+
+### Removing Peers
+
+Removing a peer node is a necessary step when you no longer want the node in the
+cluster. This could happen if the node is rotated for a new one, the hostname
+permanently changes and can no longer be accessed, you're attempting to shrink
+the size of the cluster, or for many other reasons. Removing the peer will
+ensure the cluster stays at the desired size, and that quorum is maintained. 
+
+To remove the peer you can issue a
+[remove-peer](/docs/commands/operator/raft/#remove-peer) command and provide the
+node ID you wish to remove:
+
+```shell
+$ vault operator raft remove-peer node1
+Peer removed successfully!
+```
+
+### Listing Peers
+
+To see the current peer set for the cluster you can issue a
+[list-peers](/docs/commands/operator/raft/#list-peers) command. All the voting
+nodes that are listed here contribute to the quorum and a majority must be alive
+for integrated storage to continue to operate. 
+
+```shell
+$ vault operator raft list-peers
+Node     Address                   State       Voter
+----     -------                   -----       -----
+node1    node1.vault.local:8201    follower    true
+node2    node2.vault.local:8201    follower    true
+node3    node3.vault.local:8201    leader      true
+```
+
+## Server-to-Server Communication
+
+Once nodes are joined to one another they begin to communicate using mTLS over
+Vault's cluster port. The cluster port defaults to `8201`. The TLS information
+is exchanged at join time and is rotated on a cadence. 
+
+A requirement for integrated storage is that the
+[cluster_addr](/docs/concepts/ha/#per-node-cluster-address) configuration option
+is set. This allows Vault to assign an address to the node ID at join time.
+
+## Outage Recovery
+
+### Quorum Maintained
+
+This section outlines the steps to take when a single server or multiple servers
+are in a failed state but quorum is still maintained. This means the remaining
+alive servers are still operational, can elect a leader, and are able to process
+write requests. 
+
+If the failed server is recoverable, the best option is to bring it back online
+and have it reconnect to the cluster with the same host address. This will return
+the cluster to a fully healthy state. 
+
+If this is impractical you need to remove the failed server. Usually, you can
+issue a remove-peer command to remove the failed server if it's still a member
+of the cluster.
+
+If the remove-peer command isn't possible or you'd rather manually re-write the
```

Comment:
We may want to link this section to the peers.json section just the the next section is doing using `See the section below on manual recovery using peers.json for details of the
recovery procedure`.